builtin/credential/approle/path_role_test.go

```diff
@@ -1141,7 +1141,6 @@ func TestAppRole_RoleSecretID(t *testing.T) {
 		"secret_id": "abcd123",
 	}
 	roleSecretIDReq.Data = roleCustomSecretIDData
-	roleSecretIDReq.Operation = logical.UpdateOperation
```

Comment:
This was removed as the operation never changes for this request. It's not directly related to the PR, but more so a cleanup so it can be rollbacked if it's out of scope.

Reply:
Ah, I see what you mean now